modules/client-configuration/pages/automatic-client-registration.adoc

```diff
@@ -0,0 +1,72 @@
+[[automatic-client-registration]]
+= Automatic Registration of Clients Cretaed by Terraform
+
+
+New clients created by Terraform can be automatically registered to {productname}.
+
+This can be achieved in two ways:
+
+* cloud-init based registration
```

Comment:
is this a command?  If yes:
```suggestion
* [command]``cloud-init`` based registration
```
if it is a concept, better use human understandable words `;)`